docs/Medical/Medical.md

```diff
@@ -35,7 +35,7 @@ Knowing what ACE simulates will help you know what to care about when the bullet
 
 
 #### Bleeding
-  Bleeding is caused by open wounds. The player starts with REPLACE ME Liters of blood. Blood does not regenerate on its own, but can be given by using an IV bag of Blood, Plasma, or Saline. There is no difference between the types of IV liquid in terms of gameplay or effect. FACTCHECK THIS.
+  Bleeding is caused by open wounds. The player starts with 6 Liters of blood. Blood does not regenerate on its own, but can be given by using an IV bag of Blood, Plasma, or Saline. There is no difference between the types of IV liquid in terms of gameplay or effect. FACTCHECK THIS.
```

Comment:
6 liters of blood as defined in ace_medical_engine/script_macros_medical.hpp line 49

For the second FACTCHECK I think ace_medical_engine/script_macros_medical.hpp line 63 shows the rate of change as 4.2 ml/s and in our testing giving multiple types of IV does increase the rate of change so I think that's the only thing we need to tell. But this is for medics. As for infantry guys there is (as far as I know no gameplay difference)

Reply:
Lgtm.

We'll have to look further into the IV bags, but since the non-medics use it its not very important. 
Might even be able to skip that part.